<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="2" hostname="SumedhKrishna" failures="4" tests="4" name="EDGE(failed)(failed)" time="38.046" errors="0" timestamp="2023-09-05T10:26:10 EDT">
  <testcase classname="com.tekarck.testCases.TC02_ValidLoginToSalesForce" name="loginToSalesForceAppUsingDataProviders" time="6.037">
    <failure type="java.lang.AssertionError" message="Failed to login for User Name and password :anusha@tek.com  Sumedh expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: Failed to login for User Name and password :anusha@tek.com  Sumedh expected [true] but found [false]
at com.tekarck.testCases.TC02_ValidLoginToSalesForce.loginToSalesForceAppUsingDataProviders(TC02_ValidLoginToSalesForce.java:164)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 28 stack frames]]>
    </failure>
  </testcase> <!-- loginToSalesForceAppUsingDataProviders -->
  <testcase classname="com.tekarck.testCases.TC02_ValidLoginToSalesForce" name="loginToSalesForceAppUsingDataProviders" time="5.714">
    <failure type="java.lang.AssertionError" message="Failed to login for User Name and password :anusha@tek  Sumedh@03 expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: Failed to login for User Name and password :anusha@tek  Sumedh@03 expected [true] but found [false]
at com.tekarck.testCases.TC02_ValidLoginToSalesForce.loginToSalesForceAppUsingDataProviders(TC02_ValidLoginToSalesForce.java:164)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 28 stack frames]]>
    </failure>
  </testcase> <!-- loginToSalesForceAppUsingDataProviders -->
  <testcase classname="com.tekarck.testCases.TC02_ValidLoginToSalesForce" name="loginToSalesForceAppUsingDataProviders" time="16.908">
    <failure type="java.lang.AssertionError" message="Failed to login for User Name and password :anusha@t.com  Anusha expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: Failed to login for User Name and password :anusha@t.com  Anusha expected [true] but found [false]
at com.tekarck.testCases.TC02_ValidLoginToSalesForce.loginToSalesForceAppUsingDataProviders(TC02_ValidLoginToSalesForce.java:164)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 28 stack frames]]>
    </failure>
  </testcase> <!-- loginToSalesForceAppUsingDataProviders -->
  <testcase classname="com.tekarck.testCases.TC01_LoginErrorMessage" name="validateLoginToTheApplnWithBlankPassword" time="0.745">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at com.tekarck.testCases.TC01_LoginErrorMessage.validateLoginToTheApplnWithBlankPassword(TC01_LoginErrorMessage.java:73)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 29 stack frames]]>
    </failure>
  </testcase> <!-- validateLoginToTheApplnWithBlankPassword -->
  <testcase classname="com.tekarck.testCases.TC02_ValidLoginToSalesForce" name="launchApp" time="0.0">
    <ignored/>
  </testcase> <!-- launchApp -->
  <testcase classname="com.tekarck.testCases.TC01_LoginErrorMessage" name="launchApp" time="0.0">
    <ignored/>
  </testcase> <!-- launchApp -->
</testsuite> <!-- EDGE(failed)(failed) -->
